contrib/etc/ebusd/mqtt-hassio.cfg

```diff
@@ -146,11 +146,10 @@ circuit_part = {
 # This is also an implicit field type filter as missing or empty mappings are not published at all.
 type_map-number = number
 type_map-list = string
+type_map-string = string
```

Comment:
why is string included when the PR suggests addition of date/time values?

Reply:
I don't remember. Feel free do remove it if it is not required.